contributing.md

```diff
@@ -112,6 +112,18 @@ tests/test_default.yml`
    there are any errors. If there is no problem with your contribution, the mantainer
    will merge it to the main project.
 
+### Find other images for testing
+To use the CentOS 6, CentOS 7 images, please checkout the link below:  
+
+https://cloud.centos.org/centos/7/images/CentOS-7-x86_64-GenericCloud-2003.qcow2c
+
+- `7/images/` stands for CentOS 7 images, we can also change it to `6/images/` to use
+CentOS 6 images.
+- For above qe testing cases, we prefer the image architecture to be `x86_64-GenericCloud`.
+- `2003` in `CentOS-7-x86_64-GenericCloud-2003.qcow2c` stands for image released in
+March 2020. 
+- We can use the image with extension `.qcow2` and `.qcow2c`. 
```

Comment:
Great explanation, please also add the actual URLs to the directories for CentOS 6 and 8. Also, please mention Fedora as well.

Reply:
@tyll , I do not know where to find the Fedora images

Reply:
Not sure how we should document that.  The main page for getting Fedora is https://getfedora.org/ - but it does not have links specifically for the type of cloud images we use for integration/qemu testing.  These are listed under Fedora Alternative Downloads, then under Cloud Base Images, then under https://alt.fedoraproject.org/cloud/, then under Cloud Base image for Openstack
We could also provide a link directly to the download site e.g. http://mirrors.kernel.org/fedora/releases/$FEDORA_VERSION/Cloud/x86_64/images/Fedora-Cloud-Base-$FEDORA_VERSION-$ver.x86_64.qcow2

Reply:
updated

Reply:
Even better - for Fedora, we should recommend to use the latest qcow2 images from https://kojipkgs.fedoraproject.org/compose/cloud/latest-Fedora-Cloud-31/compose/Cloud/x86_64/images/ and https://kojipkgs.fedoraproject.org/compose/cloud/latest-Fedora-Cloud-32/compose/Cloud/x86_64/images/.

Reply:
updated